icectf-2015/programming/statistics/README.md

```diff
@@ -12,7 +12,75 @@
 
 ## Write-up
 
-(TODO)
```

Comment:
Oh, I thought, since the challenge is quite obvious, that the code would be self explanatory.
I will write some text and submit again, wait for it !